src/textual/renderables/text_opacity.py

```diff
@@ -120,5 +121,5 @@ def frange(start, end, step):
 
     with Live(opacity_panel, refresh_per_second=60) as live:
         for value in itertools.cycle(frange(0, 1, 0.05)):
-            opacity_panel.value = value
+            opacity_panel.opacity = value
```

Comment:
This was an actual bug! `opacity_panel` does not have a `value` property, only `opacity`.

Reply:
This is dead code written to test the module. We should delete it.

Reply:
Will delete.